_includes/header.html

```diff
@@ -16,29 +16,26 @@
+          <li><a href="http://www.w3.org/TR/WCAG20-TECHS/">Techniques for WCAG 2.0</a> implementation guidance</li>
+          <li><a href="http://www.w3.org/TR/UNDERSTANDING-WCAG20/">Understanding WCAG 2.0</a> supporting information</li>
         </ul>
-        <p>You can customize what is included in this page by selecting from the “Filter” sidebar which Technologies, <a href="http://www.w3.org/TR/2013/NOTE-UNDERSTANDING-WCAG20-20130905/conformance.html#uc-levels-head">Levels</a> of success criteria, and Sections of techniques you want to include. Also, you can use tags to filter by use cases.</p>
+        <p>More background on these resources in provided in <a href="http://www.w3.org/WAI/intro/wcag.php">WCAG Overview</a> and <a href="http://www.w3.org/WAI/intro/wcag20">The WCAG 2.0 Documents</a>.</p>
       </div>
     </div>
     <div class="pibox">
       <div class="pibox-heading">How to Use</div>
       <div class="pibox-body">
-        <p>To customize the view, click on “Filter” in the upper right:</p>
+        <p>Select the “Filter” tab in the main menu to customize:</p>
         <ul>
-        <li><p><strong>Tags</strong> are applied to success criteria. When selecting a tag, only the criteria that have that tag associated with them are visible, the others are hidden. If you select multiple tags, every success criteria is displayed that has one or more of the tags.</p></li>
-        <li><p>With the <strong>Levels</strong> filter, it is possible to hide levels of conformance that are currently not relevant to the user, for example hide Level AAA if testing for Levels A and AA.</p></li>
-        <li><p>The <strong>Technologies</strong> filter tailors down the list of techniques for each success criterion to show only techniques for the technologies used.</p></li>
-        <li><p>By using the <strong>Techniques</strong> filter, you can by default hide sufficient and/or advisory techniques as well as failures by default. Such technique types can also be shown or hidden on a success criteria basis.</p></li>
+          <li><strong>Tags</strong> &mdash; shows only success criteria associated with the selected tags.</li>
+          <li><strong>Levels</strong> &mdash; shows only success criteria associated with the selected levels.</li>
+          <li><strong>Technologies</strong> &mdash; shows only techniques associated with the selected technologies.</li>
+          <li><strong>Techniques</strong> &mdash; shows only the types of techniques and/or failuers selected.</li>
         </ul>
-        <p><em>Note:</em> Other success criteria than the ones shown after filtering might also apply in certain circumstances.</p>
-        <p><strong>Share</strong> a link to this view by chosing the button in the top bar, or send links to specific success criteria by using the share button.</p>
+        <p><strong>Share</strong> buttons allow you to link to the customized view and to individual success criteria.</p>
```

Comment:
I suggest changing "Link to this view" to something like "Share this view" so that you are using the word "share" consistently for all the same types of share buttons.